src/main/java/seedu/anilist/ui/StatsDisplay.java

```diff
@@ -57,6 +58,28 @@
     @FXML
     private StackPane statusbarPlaceholder;
 
+    class IntegerStringConverter extends StringConverter<Number> {
+        private long mult = 100000000;
+
+        public IntegerStringConverter() {}
+
+        @Override
+        public String toString(Number object) {
+            String tickLabel = "";
+            //handle error where integer values do not show up due to rounding errors
+            if (Math.ceil(object.doubleValue()) == (Math.round(object.doubleValue() * mult) / mult)) {
+                tickLabel += (int) Math.ceil(object.doubleValue());
+            }
```

Comment:
Might I suggest this way instead (define `epsilon` on top):
```suggestion
            if (object.intValue() - epsilon <= object.doubleValue()
                && object.intValue() + epsilon >= object.doubleValue()) {
                tickLabel += object.intValue();
            } else if (object.intValue() - epsilon + 1 <= object.doubleValue()
                && object.intValue() + epsilon + 1 >= object.doubleValue()) {
                tickLabel += (object.intValue() + 1);
            }
```